hackathon_site/dashboard/frontend/src/slices/hardware/hardwareSlice.ts

```diff
@@ -68,6 +70,55 @@ export const getHardwareWithFilters = createAsyncThunk<
     }
 );
 
+export const getHardwareNextPage = createAsyncThunk<
+    APIListResponse<Hardware>,
+    void,
+    { state: RootState; rejectValue: RejectValue; dispatch: AppDispatch }
+>(
+    `${hardwareReducerName}/getHardwareNextPage`,
+    async (_, { dispatch, getState, rejectWithValue }) => {
+        try {
+            const nextFromState = hardwareNextSelector(getState()) ?? "";
+            if (nextFromState) {
+                const { path, filters } = stripHostnameReturnFilters(nextFromState);
+                const response = await get<APIListResponse<Hardware>>(path, filters);
+                return response.data;
+            }
+            // return empty response if there is no nextURL
```

Comment:
Not so sure this is a good idea. If there is no nextURL, we should just be returning nothing. You can update the return type of the thunk (line 74) to be `APIListResponse<Hardware> | null`, and handle the null case accordingly in the reducer.

This fake response is going to end up buing inserted into the redux store.

Reply:
okay, I'll try that

Reply:
do i need to test the slice for this case?

Reply:
As long as there's a test somewhere that triggers the return null case, you're good (even if it's part of the inventory tests)